proto/message/tx.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+
+package message;
+
+option go_package = "github.com/ava-labs/avalanchego/proto/pb/message";
+
+message Message {
+  oneof message {
+    Tx tx = 1;
```

Comment:
Q, are we going to add more fields other than Tx?

Reply:
We probably will at some point